### PR TITLE
Add `consumer_pool_abort_on_exception` option, defaults to false.

### PR DESCRIPTION
**Pull request by seamusabshere**

### CODE REVIEW

**Comment by seamusabshere** [comment]
requires https://github.com/ruby-amqp/bunny/pull/382

**Comment by seamusabshere** [comment]
(fails because bunny change hasn't been accepted, if/when it is, i will update this PR to require the new version of bunny)

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 No changes yet.
 
+### Configurable Consumer Thread Pool Exception Handling
```
Comment (michaelklishin) [inline CHANGELOG.md]: Perhaps the "No changes yet" line above is worth removing now.

**Comment by michaelklishin** [comment]
Bunny `2.3.0` is out.

**Comment by seamusabshere** [comment]
@michaelklishin this should be good to go

**Comment by seamusabshere** [comment]
@michaelklishin thank you

**Comment by seamusabshere** [comment]
@michaelklishin gem release planned any time soon?

**Comment by seamusabshere** [comment]
@michaelklishin ping

**Comment by michaelklishin** [comment]
@seamusabshere I'm sorry but there isn't enough stuff in master to warrant a release. I'd recommend using Hutch from `master` for now, Bundler makes that fairly trivial. When nothing shows up for a few months, I do releases to not keep contributors waiting/using master for too long.

**Comment by seamusabshere** [comment]
@michaelklishin got it

**Pull request merged**